.github/workflows/test-builds.yml

```diff
@@ -52,13 +52,19 @@ jobs:
         - jgaurora_a5s_a1
         - STM32F103VE_longer
         - STM32F407VE_black
+        - STM32F401VE_STEVAL
+        - BIGTREE_BTT002
         - BIGTREE_SKR_PRO
         - BIGTREE_GTR_V1_0
         - mks_robin
         - ARMED
         - FYSETC_S6
+        - STM32F070RB_malyan
         - malyan_M300
         - mks_robin_lite
+        - FLYF407ZG
+        - rumba32_f446ve
+        - rumba32_mks
```

Comment:
I suspect you won't want to leave all these enabled all the time, but they are useful for testing this and future changes to the platform.

Reply:
It is good to use these tests as opportunities to test features in combinations that are not covered in other tests, since more can be done in parallel. If enabling some new test environment doesn't cause new / different code to be compiled, except perhaps platform stuff, then we don't need to be redundant. One active test per HAL should sufficient.